packages/x-date-pickers/src/TimeClock/Clock.tsx

```diff
@@ -42,6 +42,10 @@ export interface ClockProps extends ReturnType<typeof useMeridiemMode>, FormProp
    * The current full date value.
    */
   value: PickerValidDate | null;
+  /**
+   * Minimal and maximal value of the clock.
```

Comment:
```suggestion
   * Minimum and maximum value of the clock.
```

According to the dictionary "minimal" is used for uncountable values.